ruma/Cargo.toml

```diff
@@ -57,6 +57,7 @@ unstable-pre-spec = [
     "ruma-client-api/unstable-pre-spec",
     "ruma-events/unstable-pre-spec",
     "ruma-federation-api/unstable-pre-spec",
+    "ruma-common/unstable-pre-spec",
```

Comment:
There are dependencies to ruma-common in other crates that expose an `unstable-pre-spec` feature. Please also add `ruma-common/unstable-pre-spec` to their unstable-pre-spec feature dependencies.

Reply:
I have added it to
 - ruma-events
 - ruma-federation-api
 - ruma-client-api